libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testAxesTableEditMessageML.ml

```diff
@@ -8,13 +8,13 @@
   
   <table><thead><tr>
   
-    <td ><b>Isin</b></td>
+    <td style="text-align:center; width:10%" ><b>Isin</b></td>
   
     <td style="text-align:center;" ><b>Bid Axed</b></td>
   
     <td style="text-align:center;" ><b>Ask Axed</b></td>
   
-    <td ><b>Creator</b></td>
+    <td style="text-align:center; width:10%" ><b>Creator</b></td>
```

Comment:
same here

Reply:
Done. Kept the String representation as like older one